public/provisioning.php

```diff
@@ -149,8 +149,11 @@
     // Add provisioning_complete variable
     $scope_data['provisioning_complete'] = '';
 
-    // Add provisioning_url_path variable
+    // Add provisioning_url_path and provisioning_url_host variables
     $scope_data['provisioning_url_path'] = $config['provisioning_url_path'];
+    if(empty($scope_data['provisioning_url_host'])) {
+        $scope_data['provisioning_url_host'] = gethostname();
```

Comment:
Can you split this change to a distinct PR?

Reply:
Done: https://github.com/nethesis/tancredi/pull/48